vhdbuilder/packer/install-dependencies.sh

```diff
@@ -352,6 +353,11 @@ for CNI_PLUGIN_VERSION in $CNI_PLUGIN_VERSIONS; do
     echo "  - CNI plugin version ${CNI_PLUGIN_VERSION}" >> ${VHD_LOGS_FILEPATH}
 done
 
+# IPv6 nftables, only Ubuntu for now
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
```

Comment:
@hbeberman - FYI when I tried testing this on a mariner nodepool I wasn't able to install nftables. Any chance we could get it added in the future?

```sh
> root@aks-mariner-38336011-vmss000000 [ / ]# yum install nftables
Loaded plugin: tdnfrepogpgcheck
No package nftables available
Error(1011) : No matching packages
```

Reply:
discussed offline, fine to scope to Ubuntu for now and follow up.